### ISSUE TITLE
#Offtopic: Do this more often

### ISSUE THREAD

**Issue by t0b1nux** [brouberol/marcel#31]
I find very interesting to do things such as this because it's a great training on how to use github and working in teams. We shoul do this more often...

**Comment by brouberol** [comment]
Agreed. That's why I've setup a CI, coverage measurement. @tgermain has suggested we integrate a sonar also. All that does not really make sense for a joke project like this one.. However, you're right, it's good training :)
